Accessibility

Danie12345 opened the pull request:
Accessibility details are now taken into account for screen readers.

### Page titles - no issues found
### Image alternatives 
- [Menu](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R30)
- [Icons](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R51-R55)
- [Placeholder](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R68)
- [Icons (bottom)](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R372-R376)
### Text headings
- [Changed `h2` for `div` due to linter errors and suggestions, plus hierarchy issues](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R372-R376)
- [Changed `h3` for `div` due to linter errors and suggestions, plus hierarchy issues](https://github.com/Danie12345/portfolio-mobile/compare/accessibility?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R349)
### Color contrast - no issues found
### Resize - no issues found
### Interaction - no issues found
### Moving content - no issues found
### Multimedia - no issues found
### Basic structure of page - no issues found

Danie12345 committed:
Fixed headers hierarchy

Made necessary changes for headers to be at the correct indentation levels.

Enekwestanley committed:
Add image text alternatives

Danie12345 committed:
Fixed linter errors and accessibility problems

Removed a few selectors in index.css, as well as fixed linter errors.

Fixed structure of index.html to maintain accessibility standards.

Small package-lock.json update upon running commands to fix a linter issue.

AmaduKamara commented on index.html:
```diff
@@ -254,7 +250,7 @@ <h3 class="project-title">Professional Art Printing Data</h3>
         </section>
         <section class="about-me decorative-background2">
-            <h2 class="h2-about-me">About me</h2>
+            <div class="h2-about-me">About me</div>
```
[ Optional Changes ] 

- [x] Kindly make this a heading text because this is meant to be the heading for the About Me section. For heading hierarchy purposes you can make it a heading 3 as you did with the other headings. 👍

AmaduKamara commented on index.html:
```diff
@@ -350,10 +346,10 @@ <h3 class="h3-about-me-outer">
         <section class="form-container">
-            <h3 class="form-text">
+            <div class="form-text">
                 I'm always interested in hearing about new projects, 
                 so if you'd like to chat please get in touch.
-            </h3>
+            </div>  
```
[ Optional Changes ] 

- [x] Kindly make this a heading text because this is meant to be the heading for the Contact Me section. For heading hierarchy purposes you can make it h3 as you did with the other headings. 👍

AmaduKamara reviewed (approved):
#Status: Approved 🟢

Hi @Danie12345,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Danie12345 committed:
Fixed optional suggestions

Replaced divs for h3 tags in index.html.

Renamed a few classes for the above changes in index.css.
